modules/serverless-create-trigger-odc.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * /serverless/event_workflows/serverless-using-brokers.adoc
+
+[id="serverless-create-trigger-odc_{context}"]
+= Creating a trigger using the Developer perspective
+
+After you have created a broker, you can create a trigger in the web console *Developer* perspective.
+
+.Prerequisites
+To create triggers using the *Developer* perspective ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console.
+* You are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created a broker and a Knative service or other event sink to connect to the trigger.
+
+.Procedure
+
+. In the *Developer* perspective, navigate to the *Topology* page.
+. Hover over the broker that you want to create a trigger for, and drag the arrow. The *Add Trigger* option is displayed.
++
+image::add-trigger-odc.png[Create a trigger for the broker]
+.. Select your sink as a *Subscriber* from the drop-down list.
++
+image::create-trigger-odc-sink.png[Select the Subscriber]
```

Comment:
Just wondering if this screenshot is truly necessary.

Reply:
removed